cmd/ethkey/utils.go

```diff
@@ -155,5 +155,7 @@ func ReadInput(message string) (string, error) {
 		return "", err
 	}
 
+	input = strings.TrimSpace(input)
```

Comment:
@kosegor does this also remove trailing characters? What happened is that, when I typed on terminal an Ethereum URL and pressed enter, the " (quotation symbol) got placed in a new line in the toml

Reply:
yes, I noticed that and it's also fixed